modules/core/src/main/scala/ch/epfl/bluebrain/nexus/rdf/Decoder.scala

```diff
@@ -64,14 +65,14 @@ object Decoder extends PrimitiveDecoderInstances with StandardDecoderInstances w
 
 trait PrimitiveDecoderInstances {
```

Comment:
Is there a `Decoder` for `FiniteDuration`? I don't seem to find it

Reply:
I haven't added it yet, will do so.